src/Administration/Resources/app/administration/src/module/sw-sales-channel/component/sw-sales-channel-measurement/sw-sales-channel-measurement.html.twig

```diff
@@ -0,0 +1,36 @@
+<sw-container
+    columns="1fr 1fr 1fr"
+    gap="0 30px"
```

Comment:
32px

Reply:
The gap here has used 30px.
https://github.com/shopware/shopware/blob/trunk/src/Administration/Resources/app/administration/src/module/sw-sales-channel/view/sw-sales-channel-detail-base/sw-sales-channel-detail-base.html.twig#L81

https://github.com/shopware/shopware/blob/trunk/src/Administration/Resources/app/administration/src/module/sw-sales-channel/component/sw-sales-channel-detail-domains/sw-sales-channel-detail-domains.html.twig#L116